queries.sql

```diff
@@ -1,3 +1,17 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * FROM animals WHERE name LIKE '%mon%';
+
+SELECT name FROM animals WHERE CAST(to_char(date_of_birth, 'YYYY') AS INTEGER) BETWEEN 2016 AND 2019;
+
+SELECT name FROM animals WHERE  neutered IS TRUE AND escape_attempts < 3;
+
+SELECT date_of_birth FROM animals WHERE name LIKE 'Agumon' OR name LIKE 'Pikachu';
+
+SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
+
+SELECT * FROM animals WHERE neutered IS TRUE;
+
+SELECT * FROM animals WHERE name NOT LIKE 'Gabumon';
```

Comment:
- [ ] Here also kindly optimize this query to something like this `SELECT * FROM animals WHERE NOT name = 'Gabumon';`

Reply:
Done